src/api/nodeRpcCall.js

```diff
@@ -1088,20 +1168,7 @@ const getLandNFTMetadataJson = async (collection_id, nft_id) => {
 }
 const setLandNFTMetadata = async (collection_id, nft_id, metadata, walletAddress) => {
   const injector = await web3FromAddress(walletAddress);
-  const api = await ApiPromise.create({
-    provider,
-    types: {
-      "Coords": {
-        "lat": "u64",
-        "long": "u64",
-      },
-      "LandMetadata": {
-        "demarcation": "BoundedVec<Coords, u32>",
-        "type": "Text",
-        "status": "Text",
-      }
-    }
-  });
+  const api = await getApi();
   /*let metadata = {
```

Comment:
i believe we should clean up this

Reply:
Out of scope for this PR. And there are other PRs that depend on this one, so such change will cause more trouble than it's worth now.